module-network.tf

```diff
@@ -131,54 +132,56 @@ output "nat_route_table_id" {
 }
 
 # Subnets
-output "subnet_ids" {
-  description = "Map of subnet ids by role for the cluster and associated resources."
-  value       = module.network.subnet_ids
-}
-output "subnet_cidrs" {
-  description = "Map of provided/calculated subnet CIDR ranges by role for the cluster."
-  value       = module.network.subnet_cidrs
-}
 output "bastion_subnet_id" {
```

Comment:
Interesting you're splitting the outputs. Is there any particular reason?

Reply:
Yeah, some lessons learned here:
- Resource Manager works better with individual outputs - typed as ocids to automatically render links.
- Output maps can be dangerous in this context - a change detected to one entry can declare the whole output "known after apply" and cause unrelated resource recreation.
- It's more lines in this file, but individual outputs are easier to work with vs. nested lookups.

Reply:
Ok, I think we'll start taking this approach of individual outputs going forward